fhir-swagger-generator/src/main/java/com/ibm/fhir/openapi/generator/FHIROpenApiGenerator.java

```diff
@@ -133,10 +135,136 @@ public static void main(String[] args) throws Exception {
             filter = createAcceptAllFilter();
         }
 
+        generateAllInOne(filter);  // outputs to all-openapi.json
+        
+        generateOneFilePerResource(filter);  // outputs to ResourceType-openapi.json
+        generateMetadataOpenApi();
+        generateBatchTransactionOpenApi(filter);
+    }
+    
+    private static void generateAllInOne(Filter filter) throws Exception {
+        JsonObjectBuilder swagger = factory.createObjectBuilder();
+        swagger.add("openapi", "3.0.0");
+
+        JsonObjectBuilder info = factory.createObjectBuilder();
+        info.add("title", "Simplified FHIR API");
+        info.add("description", "A simplified version of the FHIR API");
```

Comment:
Simplified FHIR API or "IBM FHIR Server API"? just for discussion.

Reply:
I'm fine with either.  I remember now that I had changed it (but just in this one spot) so as not to imply that the IBM FHIR Server itself can only support this simplified model.

That said, I agree we should either update it in all the spots across all swagger/openapi files, or just revert this change and go with "IBM FHIR Server API"

Reply:
yes, telling our server API is simplified doesn't sound great for me :)

Reply:
OK, I updated all the descriptions to match now.